### ISSUE TITLE
Package.loadSettings is deprecated.

### ISSUE THREAD

**Issue by alex4o** [jroesch/language-scala#17]
Store package settings files in the `settings/` directory instead of `scoped-properties/`
```
Package.loadSettings (/usr/share/atom/resources/app.asar/src/package.js:566:9)
<unknown> (/usr/share/atom/resources/app.asar/src/package.js:168:43)
Package.measure (/usr/share/atom/resources/app.asar/src/package.js:147:15)
Package.load (/usr/share/atom/resources/app.asar/src/package.js:161:12)
```

**Comment by optikfluffel** [comment]
* duplicate of #14 
* fixed in #12

**Comment by gangstead** [comment]
Fixed in `v1.0.1`
